lang/en/logstore_xapi.php

```diff
@@ -104,6 +105,8 @@
 $string['norows'] = "No rows to report";
 $string['notificationsnotenabled'] = "Notifications not enabled";
 $string['notificationtriggerlimitnotreached'] = "Notification trigger limit not reached";
+$string['user'] = 'User';
```

Comment:
remove this one?

Reply:
I was thinking it was best to have a customisable string here as this field will display the username but in the spec it uses the title "user"